pkg/swagger/swagger.go

```diff
@@ -12,6 +12,8 @@ import (
 	"github.com/Azure/ARO-RP/pkg/util/stringutils"
 )
 
+var stableAPI = "github.com/Azure/ARO-RP/pkg/api/v20200430"
```

Comment:
```suggestion
const stableAPI = "github.com/Azure/ARO-RP/pkg/api/v20200430"
```

Longer term we'll need to handle multiple stable versions, so this is not future-proof enough, but I guess it'll do for now.